docs/_form-helpers/date.md

```diff
@@ -43,10 +43,37 @@ name        | string | The name of this control
 placeholder | string | A short hint that describes the expected value
 required    | bool   | Adds `required` and `aria-required="true"` attributes
 value       | string | Specifies the value of the input
+data-datepicker | bool | Initialises or not the datepicker. Accepts `true` (default) or `false`
```

Comment:
Can you please change this back to `format` so it's consistent with the other options for this helper, the markup it outputs can still be a data attribute